projects/clr/hipamd/src/hip_code_object.cpp

```diff
@@ -511,8 +509,7 @@ hipError_t StatCO::getStatGlobalVar(const void* hostVar, int deviceId, hipDevice
   // Lazy load
   FatBinaryInfo** module = it->second->moduleInfo();
   if (*(module) == nullptr) {
-    hipError_t err = digestFatBinary(module_to_hostModule_[module], *module);
-    assert(err == hipSuccess);
+    std::ignore = digestFatBinary(module_to_hostModule_[module], *module);
```

Comment:
Same here, we should not ignore digestFatBinary Failures unless there is a special reason to like (noFatBinaryForGPU).